pkg/commands/client/option.go

```diff
@@ -32,6 +32,7 @@ type Option struct {
 	customHeaders []string
 	// this field is populated in Init()
 	CustomHeaders http.Header
+	FileSystem    bool
```

Comment:
could you add a verb to `bool`variable name

Reply:
updated.